assets/launcher/lang/sklmessages_cs_CZ.properties

```diff
@@ -90,17 +91,22 @@ settings.alert.title=Potřeba Restartovat
 settings.alert.text=Musíš restartovat launcher pro potvrzení změn.
 settings.cancel=Zrušit
 settings.save=Uložit
+
 crash.sorry=Uhoh! Vypadá to, že hra spadla. Omluvám se za nepříjemnost :(
 crash.vanilla=Pomocí kouzel a lásky se nám podařilo shromáždit některé detaily o pádu hry a vyšetříme to co nejdříve. Celou zprávu můžete vidět níže.
 crash.modded=Myslíme si, že hra může být modifikována a proto nemůžeme tuto zprávu o pádu hry příjmout. Pokud však používáte mody, prosím pošlete toto vývojářům módu, aby se na to podívali.
 crash.open=Otevřít zprávu
+
 crash.alert.title=Oops! Hra spadla
 crash.alert.text=Našli jsme ale možné řešení vašeho problému. Chcete se na to teď podívat?
 crash.alert.button.ignore=Ignorovat
 crash.alert.button.open=Otevřít stránku
+
 console.title=%s Konzole
 console.description=Tato konzole běží %s profil pro uživatele %s
+
 alert.button.confirm=Potvrdit
 alert.button.cancel=Zrušit
-about.translation=Translated by: NCPlyn (ncplyn@gmail.com)
+
+about.translation=Translated by:
```

Comment:
@NCPlyn Could you also translate this string to make the translation complete, please? It shouldn't stay in English&nbsp;:)

Also the following strings: `editor.exe`, `editor.jvm` and `settings.token` should be translated, but I think it could be hard to do so, because they contain proper nouns (like "Java" or "JVM"), so it's OK to left them as they are.

Reply:
I guess it's `Přeloženo kým:`

Reply:
You're probably right. But I still have to wait for Meka4756's answer and Waterrail's review, so the Czech translation is not the only one that delays the merge.

Reply:
Waterrail has already translated these strings.

Reply:
Waterrail has already translated these strings. I'm marking this conversation as resolved.